bin/main.rb

```diff
@@ -1,60 +1,79 @@
 #!/usr/bin/env ruby
+# rubocop:disable Metrics/BlockLength
 
-# rubocop:disable Lint/UselessAssignment
+require_relative '../lib/game_logic'
+require_relative '../lib/player'
+
+# Initialize the game and two players instances
+game = TicTacToe.new
+player1 = Player.new
+player2 = Player.new
 
 # 1. Prompt both users to give their names
 puts "Player 1, what's your name?"
-player1 = gets.chomp
```

Comment:
- [ ] You should add validations for the names of the players. It should not be like this:

![image](https://user-images.githubusercontent.com/61479648/94554035-d5e1de00-022f-11eb-9f3a-96e02245b26b.png)